cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2ValidationAndNewAPITest.java

```diff
@@ -0,0 +1,471 @@
+                        .withTargetCluster("target")
+                        .withNewSourceConnector()
+                            .withTasksMax(5)
+                            .withConfig(Map.of("sync.topic.acls.enabled", "false"))
+                        .endSourceConnector()
+                        .withNewCheckpointConnector()
+                            .withTasksMax(3)
+                            .withConfig(Map.of("sync.group.offsets.enabled", "true"))
+                        .endCheckpointConnector()
+                        .withNewHeartbeatConnector()
+                            .withTasksMax(1)
+                        .endHeartbeatConnector()
+                        .withTopicsPattern("my-topic-.*")
+                        .withTopicsExcludePattern("exclude-topic-.*")
+                        .withGroupsPattern("my-group-.*")
+                        .withGroupsExcludePattern("exclude-group-.*")
+                        .build())
+            .endSpec()
+            .build();
+
+    private final static KafkaMirrorMaker2 KMM2 = new KafkaMirrorMaker2Builder()
+            .withNewMetadata()
+                .withName("my-mm2")
+                .withNamespace("my-namespace")
+            .endMetadata()
+            .withNewSpec()
+                .withReplicas(3)
+                .withNewTarget()
+                    .withAlias("target")
+                    .withGroupId("my-mm2-group")
+                    .withConfigStorageTopic("my-mm2-config")
+                    .withOffsetStorageTopic("my-mm2-offset")
+                    .withStatusStorageTopic("my-mm2-status")
+                    .withBootstrapServers("target:9092")
+                .endTarget()
+                .withMirrors(new KafkaMirrorMaker2MirrorSpecBuilder()
+                        .withNewSource()
+                            .withAlias("source")
+                            .withBootstrapServers("source:9092")
+                        .endSource()
+                        .withNewSourceConnector()
+                            .withTasksMax(5)
+                            .withConfig(Map.of("sync.topic.acls.enabled", "false"))
+                        .endSourceConnector()
+                        .withNewCheckpointConnector()
+                            .withTasksMax(3)
+                            .withConfig(Map.of("sync.group.offsets.enabled", "true"))
+                        .endCheckpointConnector()
+                        .withNewHeartbeatConnector()
+                            .withTasksMax(1)
+                        .endHeartbeatConnector()
+                        .withTopicsPattern("my-topic-.*")
+                        .withTopicsExcludePattern("exclude-topic-.*")
+                        .withGroupsPattern("my-group-.*")
+                        .withGroupsExcludePattern("exclude-group-.*")
+                        .build())
+            .endSpec()
+            .build();
+
+    @Test
+    public void testNewLayoutValidation()    {
```

Comment:
I know this is different to all our other tests, but should we stop adding `test` at the beginning of our tests? It isn't strictly needed anymore and I don't think we should go back and change all our other tests but we could do it for new tests. It would mean when looking at the list of tests in an IDE the test names are shorter and potentially easier to scan down.

I am not strongly opposed to carrying on as we are, but just something to consider